api/apitestlib.go

```diff
@@ -33,13 +36,43 @@ type TestHelper struct {
 	SystemAdminChannel *model.Channel
 }
 
+type persistentTestStore struct {
+	store.Store
+}
+
+func (*persistentTestStore) Close() {}
+
+var testStoreContainer *storetest.RunningContainer
+var testStore *persistentTestStore
```

Comment:
If these where not global we could run MySQL and PostgreSQL in parallel.

Reply:
Not quite. There are a lot of other global variables like `utils.Cfg` and the sqlstore caches that would need to be eliminated first. And because starting up containers is too costly to do for each test (~4 seconds), we would need to add a second global instead if we were to do that. But it's my strong opinion that there's no need. The SQL stores should be tested comprehensively by the sqlstore package, and we shouldn't be tempted to make _every_ test we have a full integration test. If the sqlstore tests aren't good enough to convince us that the MySQL and PostgreSQL stores function correctly, I'll be improving them to the point that they are.